api/app/src/shared/analytics.ts

```diff
@@ -0,0 +1,41 @@
+import { PostHog } from "posthog-node";
+// import { EventMessageV1 } from "posthog-node/src/types";
+import { Config } from "./config";
+
+export interface IdentifyMessageV1 {
+  distinctId: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  properties?: Record<string | number, any>;
+}
+
+// TEMPORARY FIX - CANT EXPORT THE TYPE FROM MODULE
+export interface EventMessageV1 extends IdentifyMessageV1 {
+  event: string;
+  groups?: Record<string, string | number>; // Mapping of group type to group id
+  sendFeatureFlags?: boolean;
+  timestamp?: Date;
+}
+
```

Comment:
The typing above is copied from the node module as a temporary fix, i cant import from the module because there is a peer dependency that wasnt getting installed and i cant install it from npm

<img width="1292" alt="Screenshot 2023-03-28 at 12 49 58 PM" src="https://user-images.githubusercontent.com/36802074/228311672-25e0b82e-f199-46a9-91f0-eb8cda854829.png">

Reply:
It would be nice to open a ticket with them (if there's not one already) and have a ticket on our end, as well as reference that ticket on that comment there.

Just because this might break in the future if/when they change their API/SDK.

Reply:
https://github.com/metriport/metriport-internal/issues/493